scans/auth/auth_plan_tests.sh

```diff
@@ -89,6 +89,11 @@ for TARGET in *
 do
     if [ -d "$TARGET" ]
     then
+        if [ -f "config.disabled" ]
```

Comment:
I think this needs to be e.g. `"$TARGET/config.disabled"`

Reply:
https://github.com/zapbot/zap-mgmt-scripts/pull/242